### PR TITLE
[normal][30/5/2022]:Card

### PR DESCRIPTION
**Pull request by bisnhu-gurung123**
minor changes in styling the card

### CODE REVIEW

File: demo-app/src/app/card/card.component.html [hunk 1]
```diff
@@ -1 +1,55 @@
-<p>card works!</p>
+<div class="card-box container">
+  <div fxLayout="row" fxLayoutAlign=" center center" fxLayoutGap="760px">
```
Comment (Sonam-Choden) [inline demo-app/src/app/card/card.component.html:2]: How is this 760px used? Avoid having huge for layouts

File: demo-app/src/app/card/card.component.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  border: none;
+  margin: 16px 16px 64px;
+
+  & img {
+    width: 100%;
+    height: 100%;
+    transform: translateY(8px);
+  }
+}
+.action-card {
```
Comment (Sonam-Choden) [inline demo-app/src/app/card/card.component.scss:38]: Scss nesting should be done here

File: demo-app/src/app/card/card.component.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.container{
+  width: 1690px;
+  margin: auto;
+}
+.text {
```
Comment (Sonam-Choden) [inline demo-app/src/app/card/card.component.scss:5]: Please follow BEM class naming and the scss nesting

File: demo-app/src/app/card/card.component.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
+.container{
```
Comment (Sonam-Choden) [inline demo-app/src/app/card/card.component.scss:1]: Seeing same class repeated

File: demo-app/src/app/card/card.component.scss [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  padding: 5px;
+}
+.card {
+  height: 250px;
+  width: 310px;
+  border: none;
+  margin: 16px 16px 64px;
+
+  & img {
+    width: 100%;
```
Comment (Sonam-Choden) [inline demo-app/src/app/card/card.component.scss:33]: Use height and width attribute in image tag

File: demo-app/src/app/app.component.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
-<div class="container" fxLayout="row" fxLayoutAlign="space-between center">
-
+<app-card></app-card>
+<div class="container" fxLayout="column" fxLayoutAlign="space-between center">
```
Comment (Dhanvr) [inline demo-app/src/app/app.component.html:2]: use proper formatting "class" to be at the end

File: demo-app/src/app/app.component.scss [hunk 7]
```diff
@@ -162,7 +162,6 @@
   }
 
 }
-
 .social-link__terms {
   color: var(--color);
   font-weight: var(--bold);
```
Comment (Dhanvr) [inline demo-app/src/app/app.component.scss:167]: you can use bold directly.

File: demo-app/src/app/card/card.component.html [hunk 8]
```diff
@@ -1 +1,55 @@
...
+    <div class="text" fxLayout="row" fxLayoutAlign=" center center" fxLayoutGap="5px">
+      <p class="text-product">All Products</p>
+      <p>
+        <mat-icon class="text-product__icon">keyboard_arrow_right</mat-icon>
+      </p>
+    </div>
+  </div>
+  <div class="main_card " fxLayout="row" fxLayoutAlign=" center center" fxLayoutGap="10px">
+    <div class="card">
+      <img src="../assets/image/m1.jpg">
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.html:17]: alt?

File: demo-app/src/app/card/card.component.html [hunk 9]
```diff
@@ -1 +1,55 @@
...
+      <div class="action-card">
+        <button mat-raised-button class="product"> View Product</button>
+      </div>
+    </div>
+    <div class="card">
+      <img src="../assets/image/m3.jpg">
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.html:23]: make habit of using alt ,width and height in img tag

File: demo-app/src/app/card/card.component.html [hunk 10]
```diff
@@ -1 +1,55 @@
...
+      <div class="action-card">
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.html:24]: BEM not followed. action should be modifier. use "card__title"

File: demo-app/src/app/card/card.component.html [hunk 11]
```diff
@@ -1 +1,55 @@
...
+        <span class="action-card__title">Ski-Swiss T-shirts</span>
+        <p class="action-card__description">Lorem ipsum dolor sit amet
+        </p>
+      </div>
+    </div>
+    <div class="card">
+      <img src="../assets/image/m2.jpg">
+      <div class="action-card">
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.html:32]: use *ngFor for looping for cards

File: demo-app/src/app/card/card.component.scss [hunk 12]
```diff
@@ -0,0 +1,65 @@
+.container{
+  width: 1690px;
+  margin: auto;
+}
+.text {
+  margin: 20px 0px 0px 60px;
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.scss:6]: use mixin at least margin padding and fonts.

File: demo-app/src/app/card/card.component.scss [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+.mat-card:not([class*="mat-elevation-z"]) {
+  box-shadow: none;
+  padding: 0;
+  height: 100%;
+}
+.product {
+  width: 220px;
+  border-radius: 10px;
+  background-color: var(--higher);
+  color: var(--color);
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.scss:61]: give meaningful variable naming

File: demo-app/src/app/card/card.component.html (hunk 1 above)
Comment (Dhanvr) [reply demo-app/src/app/card/card.component.html:2]: screenshot please

File: demo-app/src/app/card/card.component.html [hunk 14]
```diff
@@ -1 +1,55 @@
-<p>card works!</p>
+<div class="card-box container">
```
Comment (bisnhu-gurung123) [inline demo-app/src/app/card/card.component.html:1]: card

File: demo-app/src/app/card/card.component.html [hunk 15]
```diff
@@ -1 +1,32 @@
-<p>card works!</p>
+<div class="box container">
```
Comment (bisnhu-gurung123) [inline demo-app/src/app/card/card.component.html:1]: demo_box

File: demo-app/src/app/card/card.component.scss [hunk 16]
```diff
@@ -0,0 +1,55 @@
+.container {
```
Comment (Dhanvr) [inline demo-app/src/app/card/card.component.scss:1]: you can use container as global

**Review by Dhanvr — APPROVED** [review]

File: demo-app/src/app/card/card.component.html [hunk 17]
```diff
@@ -1 +1,32 @@
...
+  <div fxLayout="row" fxLayoutAlign=" center center" fxLayoutGap="60px">
```
Comment (bisnhu-gurung123) [inline demo-app/src/app/card/card.component.html:2]: demo_box

**Pull request merged**